google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -146,8 +140,8 @@ def __init__(
 
         # Add the metrics headers, and instantiate the underlying GAPIC
         # client.
-        self.api = publisher_client.PublisherClient(**kwargs)
```

Comment:
Should we call out that this was removed and mark this as a breaking change?

I don't think this would have appeared in the API reference docs, so it's not quite a fully public property, but it is named like one.

Reply:
Yes, we should indeed do that, since `api` is a public attribute.